internal/campaigns/service.go

```diff
@@ -332,7 +359,7 @@ func (svc *Service) ResolveNamespace(ctx context.Context, namespace string) (str
 	if result.Data.Organization != nil {
 		return result.Data.Organization.ID, nil
 	}
-	return "", errors.New("no user or organization found")
+	return "", errors.New("failed to resolve namespace `" + namespace + "`: no user or organization found")
```

Comment:
I don't know if you know about `%q` yet, but in case you don't: you're in luck!

```suggestion
	return "", fmt.Errorf("failed to resolve namespace %q: no user or organization found", namespace)
```

Reply:
I did know about it; just didn't have a strong preference between the two in this case.